readme.md

```diff
@@ -236,6 +236,7 @@ Click the type names for complete docs.
 
 ### Miscellaneous
 
+- [`GlobalThis`](source/global-this.d.ts) - A supertype for custom subtypes of `globalThis`. Useful to declare global symbols that are targeted to a narrow scope without casting through `unknown`.
```

Comment:
Would be great if the description could be a single sentence and a little bit more succinct. The more in-depth description can be in the docs for the type.